cmd/goal/commands.go

```diff
@@ -220,6 +221,16 @@ var reportCmd = &cobra.Command{
 	},
 }
 
+var protoCmd = &cobra.Command{
+	Use:   "protocols",
+	Short: "",
+	Long:  "Dump known consensus protocols as json to stdout.",
+	Args:  validateNoPosArgsFn,
+	Run: func(cmd *cobra.Command, args []string) {
+		os.Stdout.Write(protocol.EncodeJSON(config.Consensus))
```

Comment:
mb also `-o` to dump as msgp

Reply:
I think that @pzbitskiy is a good idea, but let's finish this one first and consider to do the other in a separate PR.